config/routes.rb

```diff
@@ -239,8 +239,10 @@
     resource :session, only: %i[create destroy] do
       post 'otp_create', to: 'sessions#otp_create', as: :otp_create
       post 'webauthn_create', to: 'sessions#webauthn_create', as: :webauthn_create
+      post 'webauthn_full_create', to: 'sessions#webauthn_full_create', as: :webauthn_full_create
```

Comment:
Nit: could we either explain somewhere or update the names of these actions to better capture what exactly they are and what's different between them? Why isn't `webauthn_create` as full as `webauthn_full_create?`.